package.json

```diff
@@ -30,7 +30,7 @@
 		"@changesets/changelog-github": "^0.4.5",
 		"@changesets/cli": "^2.22.0",
 		"@types/jest": "^28.1.6",
-		"@types/node": "^16.11.11",
+		"@types/node": "^20.0.0",
```

Comment:
Should we be on types/node 20? Is this PR bumping the minimum supported version of Node to 20?

Reply:
Could be a Miniflare3 requirement, and a major version bump would be the time to do it.

Reply:
I think I'd prefer we continue to add shims? If we bump to node 20 in the types things like using global fetch will only be caught in CI.

Reply:
Fair enough 😂